### ISSUE TITLE
Unexpected expression items in DB

### ISSUE THREAD

**Issue by joeflack4** [jhu-bids/TermHub#998]
## Overview
Siggie found a sample case (non-draft cset, w/ ID = `237769358`) where a version item (concept ID = `4143008`) was found in `concept_set_version_item` but not in `concept_set_members`. This should only happen if the cset is a draft.

But this isn't the case. It's not marked as a draft in our DB or in the enclave. What's more, there is also no entry for this expression item for this cset in the enclave (in `concept_set_version_item_rv_edited`).

## Investigation
Joe looked and there are actually 152 such expression items. They've now been removed from the DB, though they are backed up here as JSON: [deletions.json](https://github.com/user-attachments/files/17928840/deletions.json)

Also checked to see if `isExcluded=true` for this expression, and that was not the case. Scrolling through the JSON, it appears many / most / all of them are set to `isExcluded=false`.

Checked to make sure that the 11 expression items showing up for this in the enclave do still appear in the database, even after that deletion operation, and they do, which is good.

Timing: The cset was created on Friday, Nov 22. Worked on this PR to handle expression item additions/deletions for drafts on Sunday the 24th. So I don't think there was anything weird going on with the refresh for this cset (or in general from what Joe can see).

I wasn't able to check the initial logs where this cset was fetched. That would be a lot of work, as it run every 20 minutes and I'd have to hunt.

`fetch_audit` entry for `237769358`: 
- Initial fetch datetime: `2024-11-22 17:07:46.125208+00`
- Successful fetch (after finalization) datetime: `2024-11-22 17:23:58.537088+00`
- Comment: _Draft cset at time reported. Fetched 0 members after fetching 156 items.; Success result: Found members_

## The cause
This is very ironic. This is a perfect case of why the following PR was needed:
- #987

Ironic because we're just noticing this as soon as I finally got around to doing it.

Basically, this cset is v7. v6 had 156 items. When it was a draft, it was fetched by TermHub and all 156 initial items, inherited from v6, were imported. Then, Siggie added and deleted some expressions. However, because #987 was not yet in place at the time, the added expressions were not added, and the deleted expressions were not deleted. This has no effect on the concept set members, which were imported correctly when the cset was finalized.

## What next
If time, Joe will work on a new issue to identify all cases where expression items are out of sync and fix them.

**Comment by joeflack4** [comment]
Addressed by:
- Finding the root cause (see "the cause" in the issue OP)
- Running `fetch_failures_0_members.py` with params `--version 237769358 --force`, which performed the synchronization operation.
